src/components/Header/Logo.js

```diff
@@ -0,0 +1,42 @@
+import React, { Fragment } from 'react'
+import styled from 'styled-components'
+import logo from '../../images/logo_animated.gif'
+import logoDesign from '../../images/design-logo.svg'
+import LogoEngComponent from '../../images/eng-logo-component'
+const HiddenText = styled.h1`
```

Comment:
Ooo, I hadn't spotted this in the codebase before! This is what you were talking about, in terms of an `h1` "for SEO purposes"? :+1: